src/packageurl/contrib/purl2url.py

```diff
@@ -304,6 +304,20 @@ def build_golang_repo_url(purl):
         return f"https://pkg.go.dev/{namespace}/{name}"
 
 
+@repo_router.route("pkg:cocoapods/.*")
+def build_cocoapods_repo_url(purl):
+    """
+    Return a CocoaPods repo URL from the `purl` string.
+    """
+    purl_data = PackageURL.from_string(purl)
```

Comment:
Please put this in a try/except block, this may fail

Reply:
@TG1999 we want to keep this function the same as other functions and not catch anything.